exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/WebServer.java

```diff
@@ -296,12 +302,19 @@ private ConstraintSecurityHandler createSecurityHandler() {
   }
 
   public int getPort() {
-    if (embeddedJetty == null || embeddedJetty.getConnectors().length != 1) {
+    if (!isRunning()) {
       throw new UnsupportedOperationException("Http is not enabled");
     }
     return ((ServerConnector)embeddedJetty.getConnectors()[0]).getPort();
   }
 
+  public boolean isRunning() {
```

Comment:
Method code can be simplified to one line.

Reply:
Agreed